openhands/runtime/__init__.py

```diff
@@ -1,25 +1,79 @@
+from dataclasses import dataclass
+
 from openhands.runtime.e2b.sandbox import E2BBox
 
 
+@dataclass
+class RuntimeInfo:
+    module: str
+    class_name: str
```

Comment:
Maybe we can use pydantic BaseModel? I'm thinking about gradually switch to pydantic over dataclass going forward 🤔

Reply:
Will change that now.